op-chain-ops/genesis/setters.go

```diff
@@ -92,7 +92,13 @@ func WipePredeployStorage(db vm.StateDB) error {
 		}
 
 		log.Info("wiping storage", "name", name, "address", *addr)
+
+		// We need to make sure that we preserve nonces.
+		oldNonce := db.GetNonce(*addr)
 		db.CreateAccount(*addr)
```

Comment:
We should also preserve balance? I don't think any of them should have balance but just in case

Reply:
Yeah good call.